applications/ParticleMechanicsApplication/custom_elements/updated_lagrangian_UP.cpp

```diff
@@ -415,9 +416,9 @@ void UpdatedLagrangianUP::CalculateAndAddRHS(
     const double& rIntegrationWeight,
     const ProcessInfo& rCurrentProcessInfo)
 {
-    rVariables.detF0   *= rVariables.detF;
-    double determinant_F = rVariables.detF;
-    rVariables.detF = 1; //in order to simplify updated and spatial lagrangian
+    //rVariables.detF0   *= rVariables.detF;
+    //double determinant_F = rVariables.detF;
+    //rVariables.detF = 1; //in order to simplify updated and spatial lagrangian
 
```

Comment:
why is the update not needed here any more?

Reply:
In the version I studied, this change was not required at all. Moreover, given the change done here and the values defined in the classes CalculatePUCoefficient() and CalculatePUDeltaCoefficient() the resulting formulation is the same as the modified, but seemed to me more elegant since resulting directly from the discretization of the mixed formulation presented in Nonlinear Solid Mechanics: An Continuum approach for Engineer, Chapter 8.